scripts/entrypoint.sh

```diff
@@ -63,16 +76,23 @@ function setup_master_db() {
       docker_temp_server_start
     fi
     init_postgres_conf
+    if [[ ${BACKUPS^^} == TRUE && -n ${STORAGE_BUCKET} && -n ${GCP_CREDENTIALS} ]]; then
+      export ARCHIVE_COMMAND="/usr/local/scripts/walg_caller.sh wal-push %p"
+    else
+      export ARCHIVE_COMMAND="cd ."
```

Comment:
What's this line for and why is this whole condition not just in the same condition around line 24?

Reply:
Different evaluations. Condition on line 24 is just to evaluate that if BACKUPS is set to true the other 2 variables needed to take backups are also set. That other condition on line 79 is needed to set ARCHIVE_COMMAND which determines where the archive logs will be stored. That command is needed to configure HA replication. Archives have to go either to GCS or be stored locally. In that second condition, evaluating BACKUPS would suffice but I added the other two variables for extra security in case they are empty.

Reply:
Just feels like that's duplication and should be merged.

I didn't mean what does the archive command do, I meant, why `cd .`

Reply:
The `cd .` just states that the WAL logs will be stored locally on a folder called `pg_wal` under the PGDATA directory path

Reply:
Thanks. Could you please add a short comment on the readme explaining the archive command and what setting it to `cd .` is important here.

Reply:
I added that as a note on line 45: `Note: HA MASTER instances with BACKUPS disabled will store WAL logs locally on the `pg_wal` folder under the PGDATA directory path.`
Is that fine or should I elaborate a bit more?

Reply:
I was thinking more of an explanation of why we set it to `cd .` specifically because it's unusual and if it just needs to be something, why is it not `echo "archiving set to ${ARCHIVE_FOLDER}`

Reply:
Updated